src/main/java/io/pinecone/PineconeIndexOperationClient.java

```diff
@@ -66,10 +77,10 @@ private Request buildRequest(String method, String path, String header, RequestB
                 .addHeader(ACCEPT_HEADER, header)
```

Comment:
**nitpick / question:** What do you think about updating the `buildRequest()` signature to rename `header` to something more specific like `acceptsHeader`?

Very minor but given that we're setting multiple headers when building the request, I feel like it might be nice to be specific.

Reply:
Done!